### PR TITLE
attach debt information

### PR DESCRIPTION
**Pull request by simonnineteen90**
https://eaflood.atlassian.net/browse/SFI-1446

### CODE REVIEW

File: app/payment-request/process-payment-request.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const getExistingPaymentRequest = require('./get-existing-payment-request')
 const savePaymentRequest = require('./save-payment-request')
 const saveInvoiceLines = require('../inbound/invoice-lines')
 const updateQualityCheck = require('../inbound/quality-checks')
+const attachDebtInformation = require('../debt/attach-debt-information')
```
Comment (AbidemiAdio) [inline app/payment-request/process-payment-request.js:6]: consider : const { attachDebtInformation } = require('../debt')

File: test/integration/local/payment-request/attach-debt-information.test.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
+const db = require('../../../../app/data')
+const { attachDebtInformation } = require('../../../../app/debt')
+const { checkDebts } = require('../../../../app/debt')
```
Comment (AbidemiAdio) [inline test/integration/local/payment-request/attach-debt-information.test.js:3]: Consider : const { attachDebtInformation, checkDebts } = require('../../../../app/debt')

File: app/debt/index.js [hunk 3]
```diff
@@ -1,9 +1,13 @@
 const getDebts = require('./get-debts')
 const getDebtsCount = require('./get-debts-count')
+const checkDebts = require('./check-debts')
+const attachDebtInformation = require('./attach-debt-information')
 const saveDebt = require('./save-debt')
```
Comment (AbidemiAdio) [inline app/debt/index.js:5]: Consider adding : const saveDebtData = require('./save-debt-data')

File: app/debt/index.js [hunk 4]
```diff
@@ -1,9 +1,13 @@
...
 
 module.exports = {
   getDebts,
   getDebtsCount,
+  checkDebts,
+  attachDebtInformation,
   saveDebt
```
Comment (AbidemiAdio) [inline app/debt/index.js:12]: then add : ,
saveDebtData

File: test/integration/local/payment-request/attach-debt-information.test.js (hunk 2 above)
Comment (simonnineteen90) [reply test/integration/local/payment-request/attach-debt-information.test.js]: updated as suggested

File: app/debt/index.js (hunk 3 above)
Comment (simonnineteen90) [reply app/debt/index.js:6]: updated as suggested

**Review by AbidemiAdio — APPROVED** [review]

**Pull request merged**